turtle_tf2_cpp/src/turtle_tf2_listener.cpp

```diff
@@ -34,6 +34,9 @@ class FrameListener : public rclcpp::Node
   FrameListener()
   : Node("turtle_tf2_frame_listener")
   {
+    turtle_spawned_ = false;
+    transform_available_ = false;
+
```

Comment:
```suggestion
  : Node("turtle_tf2_frame_listener"),
    turtle_spawned_(false),
    transform_available_(false)
  {
```

Reply:
Updated in https://github.com/ros/geometry_tutorials/pull/53/commits/5b44b06d2e1d53678c7c602ca5d7d36103c70dae